AprilSylph/XKit-Rewritten
# Classic Search: Use soft navigation

### Description


This uses the new soft navigation hook to navigate to tagged pages when classic search is not in new-tab mode.

Resolves #858.

### Testing steps


Enable classic search and disable new tab mode. Enter a search term in the search bar. The tagged page should appear quickly.
